scm/driver/codecommit/testdata/webhooks/branch_create.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@bradrydzewski I wanted to check whether branch create webhook will come under branch hook or push hook. Codecommit does not send both push & branch create trigger when a branch is created.